### ISSUE TITLE
Improve visibility of the Show in File Manager button in the project manager

### ISSUE THREAD

**Issue by WeaselOnaStick** [godotengine/godot-proposals#8875]
### Describe the project you are working on

applies to any project

### Describe the problem or limitation you are having in your project

Currently there's no way to easily navigate to the project folder within project manager. It only displays the path, but provides no way to actually opening that folder in file explorer. 

### Describe the feature / enhancement and how it helps to overcome the problem or limitation

Add "Open in file manager"/"Show in Explorer" button within the project manager

### Describe how your proposal will work, with code, pseudo-code, mock-ups, and/or diagrams

?

### If this enhancement will not be used often, can it be worked around with a few lines of script?

probably not?

### Is there a reason why this should be core and not an add-on in the asset library?

Because it's ridiculous you can't even copy the displayed path. You have to either OCR it and the paste into file explorer, or open the project, wait for it to load, then click on res folder and "Open in file manager"

**Comment by KoBeWi** [comment]
The folder icon is a button.

**Comment by WeaselOnaStick** [comment]
> The folder icon is a button.

Huh, you're right. I still feel like it should be a button on the side, to be consistent with other buttons and to just be more obvious. So I'll leave the proposal open

**Comment by quirkylemon** [comment]
do you mean something along these lines? just done properly 
I could make this tomorrow if it is ok.

![Mockup](https://github.com/godotengine/godot-proposals/assets/43453305/d3aaf8e5-cfea-41ec-afd4-3149851c8a00)

**Comment by WeaselOnaStick** [comment]
> do you mean something along these lines? just done properly I could make this tomorrow if it is ok.
> 
> ![Mockup](https://private-user-images.githubusercontent.com/43453305/297270469-d3aaf8e5-cfea-41ec-afd4-3149851c8a00.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU3MjU0NjIsIm5iZiI6MTcwNTcyNTE2MiwicGF0aCI6Ii80MzQ1MzMwNS8yOTcyNzA0NjktZDNhYWY4ZTUtY2ZlYS00MWVjLWFmZDQtMzE0OTg1MWM4YTAwLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMjAlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTIwVDA0MzI0MlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTUxZGEzNjkxMDM0NWNkMDk1ZjRkZjRkZTM3MGY5MDc1NjIyOTczODFiYWU1NGNlOWEyZWViYjg5ODNlNmE1OTEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.pRaZR4yqMk6g7Gchvnx_nDFTUgSd2S4AwgFF0ntQekI)

Yeah that's what I meant. I was thinking maybe there should also be a button for deleting the project files from disk, or maybe it should be part of the existing "delete" button's prompt. Right now you still have to manually go to project folder and delete the files (+ you still have to remove the project from the list afterwards)

**Comment by Arnklit** [comment]
I was just about to log a proposal for this myself as I didn't think there was an option for this either. I'd definitely suggest making that folder icon @KoBeWi pointed out, white instead and maybe even style it further as a button, it does not look interactable, it looks greyed out.

**Comment by SheepCommander** [comment]
THE FOLDER ICON IS A BUTTON?????? WHATTTTTT???????

tbh so many things in godot are like this, the workflow actually usually _is_ there it's just not obvious unless you get desperate and start clicking everything even if it does not conceivably look like it's meant to be interactble

**Comment by Calinou** [comment]
The button was originally made more visible in https://github.com/godotengine/godot/pull/45642, but this border was lost as buttons no longer have borders by default.

I guess we can improve this by giving this button a special background color that is always visible on the background here.
